[CAM-7553] - Review all CAM KB links to login-dashboard (#2549)
CenturyLinkCloud/PublicKB: Cloud Application Manager/Getting Started/getting-started-login-quick-tour.md

* [Quick tour](#quick-tour)

 ### Overview

>  This article makes a quick tour on login page and the panel after login to the platform in Applications site.

Reviewer: Hey @julio-castanar, this article is completely outdated, so please review it fully. It would require more changes, just follow the article and change the comments that do not correspond to the actual behavior. Thanks!

### Quick Tour

> When you log in, at the top you will see the global navigation bar. It has three sections: Service, Site and Context. 

Reviewer: ```suggestion
When you log in, at the top you will see the global navigation bar. It has three selectors: Service, Site and Context switcher. 
```

> Notice you are in the **Applications site**. In the context section you can access to **Workspaces**. 

Reviewer: ```suggestion
Notice you are in the **Applications site**. In the context switcher you can access the scope you want to work with, by selecting one among the **Workspaces**, **Cost Centers** or **Organizations** you have access to. 
```

> * Service   ( Cloud Application Manager )

Reviewer: ```suggestion
* **Service**:  Cloud Application Manager
```

> * Site      ( Applications site )

Reviewer: ```suggestion
* **Site**: Applications site
```

> * Context   ( Organization, Cost Centers or Workspaces )

Reviewer: ```suggestion
* **Context**: Organization, Cost Center or Workspace selected
```

> On the left side, a menu guides you to the main areas of Cloud Application Manager

Reviewer: ```suggestion
On the left side, a menu guides you to the main areas of Cloud Application Manager, Applications site:
```

> * Dashboard

Reviewer: Please, use **bold** for all these bullet points

* Instances
* Boxes
* Providers
* Catalog
* Activity

> On the right side of the top bar, you will find help and your account menus.

Reviewer: ```suggestion
On the right side of the top navigation bar, you will find the links to **Help** and **My Account** menus.
```

> * Your Account

Reviewer: ```suggestion
* **My Account**
```

> * Help

Reviewer: ```suggestion
* **Help**
```

___

**Dashboard**

![CAM workspace panel](../../images/cloud-application-manager/getting-started-login-8.png)

> It is a global vision of the content and uses of the workspace. There you can see the number of the instances, boxes or providers and their status. In addition, there are two charts with the uses by day, week or month.

Reviewer: ```suggestion
The Dashboard page displays a global vision of the content and uses of the current scope. There you can see the number of instances, boxes or providers it contains. In addition, there are two charts with the uses by day, week or month.
```
